src/jobs/lint-app.yml

```diff
@@ -0,0 +1,9 @@
+description: >
+  This job will lint the application based on what language the application is written in.
```

Comment:
I would love to have a meeting to understand how this will be implemented for multiple languages.

Reply:
Sounds good